app/Config/Exceptions.php

```diff
@@ -45,4 +45,16 @@ class Exceptions extends BaseConfig
 	 * @var string
 	 */
 	public $errorViewPath = APPPATH . 'Views/errors';
+        
+	/**
+	 * --------------------------------------------------------------------------
+	 * HIDE FROM DEBUG TRACE
+	 * --------------------------------------------------------------------------
+	 * Any data that you would like to hide from the debug trace.
+	 * In order to specify 2 levels, use "/" to separate.
+         * ex. ['server', 'setup/password', 'secret_token']
```

Comment:
What is `'server'` in the example? What is *level* ?
What does this feature hide?
Does this feature really work?